install/kubernetes/helm/istio/templates/crds.yaml

```diff
@@ -1,6 +1,6 @@
-# {{ if or .Values.global.crds (semverCompare ">=2.10.0-0" .Capabilities.TillerVersion.SemVer) }}
+{{- if and (.Values.global.crds) (semverCompare ">=2.10.0-0" .Capabilities.TillerVersion.SemVer) }}
```

Comment:
its a shame semverCompare doesn't work in `template` mode.  Perhaps we could add another key "override_helm_version"...?

We have a lot of folks depending on us maintaining the `template` workflow, so that is mandatory.